datadog/api/api_client.py

```diff
@@ -174,11 +174,13 @@ def submit(cls, method, path, api_version=None, body=None, attach_host_name=Fals
                 except ValueError:
                     raise ValueError('Invalid JSON response: {0}'.format(content))
 
-                if response_obj and 'errors' in response_obj:
-                    # suppress ApiError when specified and just return the response
-                    if not (suppress_response_errors_on_codes and
-                            result.status_code in suppress_response_errors_on_codes):
-                        raise ApiError(response_obj)
+                # response_obj can be a bool and not a dict
+                if not isinstance(response_obj, bool):
```

Comment:
just go with `isinstance(response_obj, dict)`?

This is going to crash if  `response_obj` ends up being a e.g. `int`.